### ISSUE TITLE
the Inconsistency between training test and the test.py

### ISSUE THREAD

**Issue by maowayne123** [open-mmlab/mmpose#998]
i enable the fp16 for training, but i got a slightly different result between training test and the testing by the test.py. there is a about 0.08ap (73.30 vs 73.21). do you have any idea? thanks

**Comment by jin-s13** [comment]
Synchronization of BatchNorm's buffer (running_mean and running_var) is not supported in the DDP of pytorch, which may cause the inconsistent performance of models in different ranks, so we broadcast BatchNorm's buffers of rank 0 to other ranks to avoid this.

https://github.com/open-mmlab/mmaction2/blob/master/mmaction/core/evaluation/eval_hooks.py#L365-L371

Maybe we should fix this problem as in mmaction2. @ly015

**Comment by maowayne123** [comment]
thanks a lot! let me check

**Comment by maowayne123** [comment]
so if i need to fix this, i needs to retrain my model?
 thanks

**Comment by maowayne123** [comment]
it is possible to fix this bu applying the syncbn?

**Comment by maowayne123** [comment]
it is possible to fix this bug by applying the syncbn?

**Comment by maowayne123** [comment]
i tested with sycnbn, there is still about 0.01ap difference, but i think that is normal, so syncbn seems can solve this.
